EIPS/eip-3475.md

```diff
@@ -357,65 +454,107 @@ e.g.
+        uint256 nonceId;
+        uint256 _amount;
+    }
+```
+Where
+
+The "classId" is the class id of the bond.
+
+The "nonceId" is the nonce id of the given bond class. This param is for distinctions of the issuing conditions of the bond.
+
+The "_amounts" is the list of amounts of the bond that the spender is approved for.
+
+
+**AMM optimization**: One of the most obvious use cases of ERC 3475 is the multilayered pool. The early version of AMM uses a separate smart contract and an ERC 20 LP token to manage a pair. By doing so, the overall liquidity inside of one pool is significantly reduced and thus generates unnecessary gas spent and slippage. Using ERC 3475, one can build a big liquidity pool with all the pairs inside (thanks to the presence of the data structures consisting of the liquidity corresponding to the given class and nonce of bonds). Thus by knowing class and nonce of the bonds , the liquidity can be represented as the percentage of a given token pair for the owner of the bond in the given pool. Effectively, the ERC20 LP token (represented by unique smart contract in the pool factory contract ) is converted, in this case, to ERC 3475 (Multiple Callable Bonds) token which is consolidated into single pool.
+
 
-**AMM optimization**: One of the most obvious use cases of ERC 3475 is the multilayered pool. The early version of AMM uses a separate smart contract and an ERC 20 LP token to manage a pair. By doing so, the overall liquidity inside of one pool is significantly reduced and thus generates unnecessary gas spent and slippage. Using ERC 3475, one can build a big liquidity pool with all the pairs inside. Then, with class and nonce, represent what percentage of a given token this address holds. Effectively, the ERC20 LP token is converted, in this case, to ERC 3475 (Multiple Callable Bonds) token.
 
 
 ## Backwards Compatibility
 
-ERC-3475 contract is not compatible with contracts that don't have an ERC-3475 interface built in. This requires the existing contract to upgrade with ERC-3475 interface. The receiving of ERC-3475 Bond need the implementation of ERC-3475 interface in the receiver contract.
+ERC-3475 contract is  compatible with contracts that  inherit  ERC-3475 interface .Its valid for both the issuer and receiver of the bonds , the client EOA needs the IERC3475 interface.
 
-However any existing ERC-20 token contract can issue their ERC-3475 bond, by giving the minting role to a bank contract with ERC-3475 interface built in. The implementation of This can be found in our Use Cases.
+However any existing ERC-20 token contract can issue their ERC-3475 bond, by delegating the  the minting role to a bank : orchestrator contract with ERC-3475 interface built in. The implementation of This can be found in our reference implementation.
 
 To ensure the reading of transactions, `"Issued"` ,`"Redeemed"` ,`"Burned"` ,`"Transfer"`, Events cited above MUST be emitted when such transaction is passed. 
```

Comment:
```suggestion
```

Shouldn't be in backwards compatibility (except that Transfer may be backwards incompatible)

Reply:
could you elaborate more on that please ? sorry i am not able to see the rendering of the changes that you are mentioning in the IDE / github integration.